core/transport/pom.xml

```diff
@@ -133,6 +134,23 @@
         </executions>
       </plugin>
 
+      <!-- Proto lock for preventing backwards compatibility issues -->
```

Comment:
please add definition of this artifact dependency  to `root/pom.xml` under 
`<dependencyManagement>`
https://github.com/Alluxio/alluxio/blob/0d149a8d397ac8c1b0092d390c85956706399ff2/pom.xml#L199
like
```
      <dependency>
        <groupId>com.salesforce.servicelibs</groupId>
        <artifactId>proto-backwards-compatibility</artifactId>
        <version>1.0.5</version>
      </dependency>
```

Then in this `core/transport/pom.xml`, you only need to refer to the artifact without specifying the version
```
      <plugin>
        <groupId>com.salesforce.servicelibs</groupId>
        <artifactId>proto-backwards-compatibility</artifactId>
        <configuration>
          <protoSourceRoot>${basedir}/target/classes/</protoSourceRoot>
        </configuration>
        <executions>
          <execution>
            <goals>
              <goal>backwards-compatibility-check</goal>
            </goals>
          </execution>
        </executions>
      </plugin>
```

Reply:
Thanks for the tip. I've updated the PR.